prover/src/lib.rs

```diff
@@ -210,18 +211,12 @@ pub trait Prover {
         // execution of the computation for the provided public inputs.
         let air = Self::Air::new(trace.get_info(), pub_inputs, self.options().clone());
 
-        // make sure the specified trace is valid against the AIR. This checks validity of both,
-        // assertions and state transitions. we do this in debug mode only because this is a very
-        // expensive operation.
-        #[cfg(debug_assertions)]
-        trace.validate(&air);
-
```

Comment:
Do we want to add this back in now that the validation is fixed?

Reply:
This is present after. Otherwise it would have required twice a check on has_raps so I put it after some operations. But the impact is not too important on debug mode anyway

Reply:
Ah, yes. I saw that later. In this case, the one thing that can be changed is the double comments (`// //`) above it.